### PR TITLE
bugfix/ST-313-cannot-make-success-payment-if-previous-payment-was-declined

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by pgs-gmartowski** [comment]
@james-simpson-ST I cannot approve my pull request, so I give 👍 here :)
Now I can fully focus on migrating animated card.

File: example/index.html [hunk 1]
```diff
@@ -99,24 +99,13 @@ <h1 class="example-form__title">
...
-              'line-height-input': '12px'
+              'background-color-input': 'AliceBlue'
             },
             securityCode: {
-              'background-color-input': '#FFF333',
-              'background-color-input-error': '#f8d7da',
-              'color-input-error': '#3358FF',
-              'font-size-input': '20px',
-              'line-height-input': '20px'
+              'background-color-input': 'AliceBlue'
```
Comment (pgs-gmartowski) [inline example/index.html]: info: to get rid of the annoying test fields styling :/

File: src/core/classes/CommonFrames.class.ts [hunk 2]
```diff
@@ -187,6 +190,7 @@ class CommonFrames extends RegisterFrames {
    */
   private _onTransactionComplete(data: any) {
     if ((this._isTransactionFinished(data) || data.errorcode !== '0') && this._submitCallback) {
+      this._validation.blockForm(false);
```
Comment (pgs-gmartowski) [inline src/core/classes/CommonFrames.class.ts]: info: here is the place where must unblock form (we've got here fixes for both 313 and 316)

**Comment by james-simpson-ST** [comment]
Could this be applied to the release branch instead of develop and then merged back? I think we will want to push this out faster so don't want to have to verify all the other tasks before we can release this

**Comment by pgs-gmartowski** [comment]
> @pgs-gmartowski Could this be applied to the release branch instead of develop and then merged back? I think we will want to push this out faster so don't want to have to verify all the other tasks before we can release this

yup, sure, unfortunately we have an issue with translation feature here (automated tests are failing), perhaps I should revert these changes (which are currently in develop) and then we can make hotfix.

**Comment by james-simpson-ST** [comment]
> > @pgs-gmartowski Could this be applied to the release branch instead of develop and then merged back? I think we will want to push this out faster so don't want to have to verify all the other tasks before we can release this
> 
> yup, sure, unfortunately we have an issue with translation feature here (automated tests are failing), perhaps I should revert these changes (which are currently in develop) and then we can make hotfix.

Yeah that makes sense, create a hotfix branch and we can then merge from hotfix into release/master and develop branches

**Pull request closed without merging**